lib/src/ast_extensions.dart

```diff
@@ -131,27 +131,21 @@ extension ExpressionExtensions on Expression {
   /// example, in an assignment, a split in the assigned value is usually
   /// indented:
   ///
-  /// ```
```

Comment:
I prefer backticks + `dart` – just to minimize the dart-specific magic. 🤷

Reply:
Using indentation is actually [the original Markdown syntax for code blocks](https://daringfireball.net/projects/markdown/syntax#precode). Backticks were a non-standard extension added by GitHub.